### ISSUE TITLE
Trigger EULA email sending via bulk checkout of already assigned assets

### ISSUE THREAD

**Issue by alfiogregorini** [grokability/snipe-it#8818]
#### Expected Behavior (or desired behavior if a feature request)

I would like to do a bulk check-out of the already assigned assets, in order to trigger the EULA email sending.

-----

#### Actual Behavior

The EULA email sending can be triggered only after a check-in and a check-out of the asset.

-----

#### Please confirm you have done the following before posting your bug report:

- [X] I have enabled debug mode 
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

-----
#### Provide answers to these questions:

- Is this a fresh install or an upgrade? Fresh install
- Version of Snipe-IT you're running - Version v4.9.4 build 4437
- Version of PHP you're running - 7.3.17
- Version of MySQL/MariaDB you're running - ?
- What OS and web server you're running Snipe-IT on - I paid for the hosted version
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) - I paid for the hosted version
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error - no error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error - no specific page
- If a stacktrace is provided in the error, include that too. - no error
- Any errors that appear in your browser's error console. - no error
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo. - no error
- Include any additional information you can find in `storage/logs` and your webserver's logs. - no logs
- Include what you've done so far in the installation, and if you got any error messages along the way. - no error
- Indicate whether or not you've manually edited any data directly in the database - no

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

https://snipe-it.readme.io/docs/getting-help

**Comment by therootadministrator** [comment]
YES, please add a button to re-send the EULA to people that haven't signed.

**Comment by snipe** [comment]
Hi there - I know this is an old issue (and we [have a cli tool for that that](https://snipe-it.readme.io/docs/unaccepted-asset-reminders#/)), but wanted to let you know that we're actively discussing best steps here in this thread. https://github.com/grokability/snipe-it/issues/16881

I'll close this one for now to keep the convo going over there.
